contracts/mocks/PolyTokenFaucet.sol

```diff
@@ -21,10 +21,18 @@ contract PolyTokenFaucet {
     event Transfer(address indexed _from, address indexed _to, uint256 _value);
     event Approval(address indexed _owner, address indexed _spender, uint256 _value);
 
+    constructor() public {
+        totalSupply_ = 1000000;
```

Comment:
multiply it by `10^decimals`.

Reply:
`decimals` is a constant in our case so it's only gonna increase the gas cost (one extra SLOAD). 

changed it anyway because it's the standard practice.